docs/client.rst

```diff
@@ -0,0 +1,22 @@
+Sphinx client cryptographic operations
+======================================
+
+create_header
+-------------
+.. autofunction:: sphinxmixcrypto.create_header
+
+create_forward_message
+----------------------
+.. autofunction:: sphinxmixcrypto.create_forward_message
+
+create_reply_block
+------------------
+.. autofunction:: sphinxmixcrypto.create_reply_block
+
+ClientMessage
+-------------
+.. autoclass:: sphinxmixcrypto.ClientMessage
+
+SphinxClient
+------------
+.. autoclass:: sphinxmixcrypto.SphinxClient
```

Comment:
yeah i agree i shouldn't have to manually type auto... for auto generated api docs. i could easily use pydoctor instead although at least one person thinks it's funny to document sphinx using sphinx. i am not amused by the namespace collision. ;-p